cocos/audio/audio-manager.ts

```diff
@@ -102,6 +102,24 @@ export class AudioManager {
             this.removePlaying(audioInfoToDiscard.audio);
         }
     }
+
+    public pause () {
+        this._oneShotAudioInfoList.forEach((info) => {
+            info.audio.stop();
+        });
+        this._audioPlayerInfoList.forEach((info) => {
+            // eslint-disable-next-line no-void
+            void info.audio.pause();
```

Comment:
Why need `void`?

Reply:
Promises must be handled appropriately or explicitly marked as ignored with the `void` operator.